### ISSUE TITLE
Does not work well with screen readers and keyboard

### ISSUE THREAD

**Issue by mstop4** [mstop4/gifcentration-2#21]
# Description

The game was not designed with screen readers and keyboard controls in mind. In particular, screen readers can read the alt text of a GIF even if the card its on hasn't been flipped, allow players to "cheat".
